tests/integration/corrupt-dump.tcl

```diff
@@ -457,11 +443,12 @@ test {corrupt payload: fuzzer findings - hash convert asserts on RESTORE with sh
     # if we don't perform full sanitization, and the next command can assert on converting
     # a ziplist to hash records, then we're ok with that happning in RESTORE too
     start_server [list overrides [list loglevel verbose use-exit-on-panic yes crash-memcheck-enabled no] ] {
-        r config set sanitize-dump-payload no
         r debug set-skip-checksum-validation 1
-        catch { r RESTORE _hash 0 "\x0D\x3D\x3D\x00\x00\x00\x3A\x00\x00\x00\x14\x13\x00\xF5\x02\xF5\x02\xF2\x02\x53\x5F\x31\x04\xF3\x02\xF3\x02\xF7\x02\xF7\x02\xF8\x02\x02\x5F\x37\x04\xF1\x02\xF1\x02\xF6\x02\x02\x5F\x35\x04\xF4\x02\x02\x5F\x33\x04\xFA\x02\x02\x5F\x39\x04\xF9\x02\xF9\xFF\x09\x00\xB5\x48\xDE\x62\x31\xD0\xE5\x63" }
-        assert_equal [count_log_message 0 "crashed by signal"] 0
-        assert_equal [count_log_message 0 "ASSERTION FAILED"] 1
+        catch {
```

Comment:
the comment above the test became outdated.

Reply:
fixed.